platform_umbrella/apps/common_core/lib/common_core/notebooks/jupyter_lab_notebook.ex

```diff
@@ -68,7 +68,13 @@ defmodule CommonCore.Notebooks.JupyterLabNotebook do
 
   batt_schema "jupyter_lab_notebooks" do
     slug_field :name
-    field :image, :string, default: "jupyter/datascience-notebook:lab-4.0.7"
+
+    field :image, :string,
```

Comment:
`defaultable_image_field` should do this for you if you wanna do it that way...

Reply:
I don't. For jupyter notebooks they should be immutable. Scientific users expect that their experiments don't change because a platform upgrades. Additionally I don't want to restrict the image to only being this one, there are a lot of jupyter notebook docker images and users can choose theirs.

My ideal would be that we have a `image_field` type that also has the existing form, while allowing an override of the full image path.